ajax/libs/flowplayer/package.json

```diff
@@ -16,5 +16,14 @@
     "type": "git",
     "url": "https://github.com/flowplayer/flowplayer"
   },
-  "license": "GPLv3"
+  "license": "GPLv3",
```

Comment:
License should be `GPL-3.0`. Ref: https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json#L162